README.md

```diff
@@ -1,96 +1,64 @@
 # KubeVirt Ansible
 
-This repository provides collection of playbooks to
-- [x] [Install KubeVirt on existing OpenShift cluster](#install-kubevirt-on-existing-cluster)
-- [ ] Deploy Kubernetes cluster on given machines and install KubeVirt
-- [x] [Deploy OpenShift cluster on given machines and install KubeVirt](#deploy-kubernetes-or-openshift-and-kubevirt)
-- [x] [Provision resources, deploy cluster and install KubeVirt](#deploy-new-kubernetes-or-openshift-cluster-and-kubevirt-with-lago)
+This repository provides a collection of playbooks to
+- [x] [Install KubeVirt on an existing OpenShift cluster](#install-kubevirt-on-existing-cluster): install-kubevirt-on-openshift.yml
```

Comment:
Fine in adding the yamls at the end, but please enclose them in backticks: `…`

I'd actually also put them in bracets (`…`)